exchanges/btse/btse_wrapper.go

```diff
@@ -608,7 +655,15 @@ func (b *BTSE) WithdrawFiatFundsToInternationalBank(withdrawRequest *withdraw.Re
 
 // GetActiveOrders retrieves any orders that are active/open
 func (b *BTSE) GetActiveOrders(req *order.GetOrdersRequest) ([]order.Detail, error) {
-	resp, err := b.GetOrders("")
+	if len(req.Pairs) == 0 {
+		return nil, errors.New("no pair provided")
+	}
+
+	formattedPair, err := b.FormatExchangeCurrency(req.Pairs[0], asset.Spot)
```

Comment:
This should range over the requested pairs as this currently will only ever do one pair